packages/models/test/fixtures/runner-config.mock.ts

```diff
@@ -37,3 +41,15 @@ export function echoRunnerConfig(
     outputFile,
   };
 }
+
+/**
+ * Use this helper as a general purpose mock with working defaults
+ * @param options
+ */
+export function esmRunnerConfig(
+  options?: Partial<EsmRunnerConfig>,
```

Comment:
The options aren't used and I don't understand the intention (`Partial` of a function type?). 

The implementation creates invalid data. Do we want that for our tests?

I would suggest that by default the runner function returns an empty array, but that I can pass in my own function as an argument.